app/vmalert/group_test.go

```diff
@@ -121,17 +131,16 @@ func TestGroupStart(t *testing.T) {
 	alert2.State = notifier.StateFiring
 	alert2.ID = hash(m2)
 
-	const evalInterval = time.Millisecond
 	finished := make(chan struct{})
 	fs.add(m1)
 	fs.add(m2)
 	go func() {
-		g.start(context.Background(), evalInterval, fs, fn, nil)
+		g.start(context.Background(), fs, fn, nil)
 		close(finished)
 	}()
 
 	// wait for multiple evals
-	time.Sleep(20 * evalInterval)
+	time.Sleep(20 * g.Interval)
```

Comment:
it is unclear what `interval` is in the rule that we read. Would be better just to override interval of parsed group with constant `1ms` value so it would be easier for reader to understand why we wait `20 * g.Interval`

Reply:
fixed